enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -66,14 +67,18 @@ const (
 	TestBusyboxImageRef = "test.buildbuddy.io/busybox"
 
 	// Image cache layout version. This should be incremented when making
-	// backwards-compatible changes to image cache storage, and older version
-	// directories can be cleaned up.
-	imageCacheVersion = "v1" // TODO: add automatic cleanup if this is bumped.
+	// backwards-compatible changes to image cache storage. Version directories
+	// which don't match this version are cleaned up automatically on startup.
```

Comment:
Mind adding a comment that this must start with a 'v' or match the versionDirRegexp lower down

Reply:
Done